package.json

```diff
@@ -301,7 +301,7 @@
     "history": "^5.3.0",
     "json-schema-to-typescript": "^10.1.5",
     "jsonpointer": "^5.0.0",
-    "next": "13.1.1",
+    "next": "13.3.0",
```

Comment:
do we need to update this now? I think we need to do it separately -- also missing `yarn.lock` changes?